src/contexts/AudioPlayerContext.tsx

```diff
@@ -89,7 +92,32 @@ export const AudioPlayerProvider: React.FC = ({ children }) => {
       if (source === 'azure') {
         setMimeType('application/x-mpegURL')
         setAudioUrl(`${source}(format=m3u8-cmaf)`)
-      } else if (source === 'cloudflare') {
+      }
+
+      if (cloudfrontPath) {
+        axios
+          .get(`${process.env.REACT_APP_LODESTAR_SERVER_ENDPOINT}/videos/${videoId}/sign`, {
+            headers: {
+              Authorization: `Bearer ${authToken}`,
+            },
+          })
+          .then(({ data }) => {
+            const {
+              videoSignedPaths: { hlsPath, dashPath, cloudfrontMigratedHlsPath },
+            } = data.result
+
+            setMimeType('application/x-mpegURL')
+            setAudioUrl(
+              hlsPath && dashPath
+                ? `${process.env.REACT_APP_LODESTAR_SERVER_ENDPOINT}/videos${hlsPath}`
+                : `${process.env.REACT_APP_LODESTAR_SERVER_ENDPOINT}/videos${cloudfrontMigratedHlsPath}`,
+            )
+          })
+
+        return
+      }
+
+      if (source === 'cloudflare') {
```

Comment:
could move `cloudfrontPath` forward
such like
if(cloudfrontPath){
...
}
if(source ==='azure'){
...
}else if (source==='cloudflare){
...
}

Reply:
I think azure should be prioritized over cloudFront because if someone is using Azure, they  wouldn't use cloudFront.